README.md

```diff
@@ -1,218 +1,180 @@
 
-At this point, your config is saved to a configuration file managed by
-`ankerctl.py`. To see an overview of the stored data, use `config show`:
+   ```sh
+   ./ankerctl.py config show
+   [*] Account:
+       user_id: 01234567890abcdef012...<REDACTED>
+       email:   bob@example.org
+       region:  eu
+   
+   [*] Printers:
+       sn: AK7ABC0123401234
+       duid: EUPRAKM-001234-ABCDE
+   ```
 
-```sh
-./ankerctl.py config show
-[*] Account:
-    user_id: 01234567890abcdef012...<REDACTED>
-    email:   bob@example.org
-    region:  eu
-
-[*] Printers:
-    sn: AK7ABC0123401234
-    duid: EUPRAKM-001234-ABCDE
-```
+> **NOTE:** 
+> The cached login info contains sensitive details. In particular, the `user_id` field is used when connecting to MQTT servers, and essentially works as a password. Thus, the end of the value is redacted when printed to screen, to avoid accidentally disclosing sensitive information.
 
-NOTE: The cached login info contains sensitive details. In particular, the
-`user_id` field is used when connecting to MQTT servers, and essentially works
-as a password. Thus, the end of the value is redacted when printed to screen, to avoid
-accidentally disclosing sensitive information.
+2. Now that the printer information is known to `ankerctl`, the tool is ready to use. There’s a lot of available commands and utilities, use a command followed by `-h` to learn what your options are and get more in specific usage instructions.
 
-Now that the printer information is known to `ankerctl`, the tool is ready to use.
+> **NOTE:**
+> As an alternative to using "config import" on the command line, it is possible to upload `login.json` through the web interface. Either method will work fine.
 
-Some examples:
+## Usage
 
-```sh
-# attempt to detect printers on local network
-./ankerctl.py pppp lan-search
+### Web Interface
 
-# monitor mqtt events
-./ankerctl.py mqtt monitor
+1. Start the webserver by running one of the following commands in the folder you placed ankerctl in. You’ll need to have this running whenever you want to use the web interface or send jobs to the printer via a slicer:
 
-# start gcode prompt
-./ankerctl.py mqtt gcode
+   Docker Installation Method:
 
-# set printer name
-./ankerctl.py mqtt rename-printer BoatyMcBoatFace
+   ```sh
+   docker compose up
+   ```
 
-# print boaty.gcode
-./ankerctl.py pppp print-file boaty.gcode
+   Git Installation Method Using Python:
 
-# capture 4mb of video from camera
-./ankerctl.py pppp capture-video -m 4mb output.h264
-```
+   ```sh
+   ./ankerctl.py webserver run
+   ```
 
-## Webserver
+2. Navigate to [http://localhost:4470](http://localhost:4470) in your browser of choice on the same computer the webserver is running on. 
+ 
+ > **Important**
+ > You’ll be prompted to upload your `login.json` file and the given the default path it should be found in your corresponding Operating System. 
```

Comment:
This is only the case if you did not previously upload your config.

Reply:
Added that caveat